src/site/markdown/getting_started.md

```diff
@@ -1,316 +1,246 @@
-are actually much more powerful due to scala's facility for 
-[implicit parameters](http://daily-scala.blogspot.com/2010/04/implicit-parameters.html).
 
+## Build Aloha from Source
 
->
-> Imports can do more than just add language features. They can *change the meaning of the language*.
->
-
-
-Imports are specified globally in CompiledSemantics, away from where the features are specified. This is by design 
-so that the same specifications can have different meaning in different problem domains.  The consequence is that 
-imports need to be used with caution, especially when referencing implicit language features.  The following example 
-illustrates this.  
-
-Imagine a third party developer wrote the following code 
-
-```scala
-package com.fictitious.company
+```bash
+git clone git@github.com:eHarmony/aloha.git
+cd aloha
+mvn clean install
+```
 
-object Ops {
-  /**
-   * @param v an explicit value
-   * @param a an implicit value to add to v 
-   * @param ring A type class representing an algebraic ring.
-   * @tparam A the type of algebraic ring
-   * @return ''v'' added to some value ''a'' in the implicit scope
-   */
-  def add[A](v: A)(implicit a: A, ring: Numeric[A]) = num.plus(v, a)
-}
+If you have Zinc installed and running, and are using Maven 3, change:
 
-object Implicits {
-  implicit val two: Int = 2 
-  implicit val four: Int = 4
-}
+```bash
+mvn clean install
 ```
 
-Let’s assume a feature specification: `add(${some.value})` and the parameter 
-`${some.value}` has an integer value, *1*. If the imports were: 
-
-- `com.fictitious.company.Ops.add`
-- `com.fictitious.company.Implicits.two`
+*to* 
 
-Then the output of the feature would be *3* in this instance.  If we changed the imports to 
+```bash
+mvn -Dcompile=incremental clean install
+```
 
-- `com.fictitious.company.Ops.add`
-- `com.fictitious.company.Implicits.four`
+## Create a VW Dataset 
 
-then the **same feature specification** would yield a value of *5*.  While this can be extremely useful, it is also
-extremely powerful.  It is imperative to understand the imports being imported into scope.
+*[eHarmony](http://www.eharmony.com)* uses [Vowpal Wabbit](https://github.com/JohnLangford/vowpal_wabbit/wiki) for many
+of its predictive tasks so Aloha provides support for VW including dataset creation and native VW model creation on the 
+[JVM](https://en.wikipedia.org/wiki/Java_virtual_machine).
 
+<span class="label">bash script</span>
 
-### Protobuf Plugin 
+```bash
+aloha-cli/bin/aloha-cli                                  \
+  -cp $(find $PWD/aloha-cli -name "*.jar" | grep dep):\
+$(find $PWD/aloha-core -name "*.jar" | grep test)        \
+  --dataset                                              \
+  -s $(find $PWD/aloha-core/src -name 'proto_spec2.json')\
+  -p com.eharmony.aloha.test.proto.Testing.UserProto     \
+  -i $(find $PWD/aloha-core/src -name 'fizz_buzzs.proto')\
+  --vw_labeled /tmp/dataset.vw
+```
```

Comment:
Oh. I didn't realize that's all you meant. Yeah. I can do that no problem.



On Sep 14, 2015, 4:38 PM, at 4:38 PM, Jon Morra <notifications@github.com> wrote:
>>  
>> -### Protobuf Plugin 
>> +```bash
>> +aloha-cli/bin/aloha-cli                                  \
>> +  -cp $(find $PWD/aloha-cli -name "*.jar" | grep dep):\
>> +$(find $PWD/aloha-core -name "*.jar" | grep test)        \
>> +  --dataset                                              \
>> +  -s $(find $PWD/aloha-core/src -name 'proto_spec2.json')\
>> +  -p com.eharmony.aloha.test.proto.Testing.UserProto     \
>> +  -i $(find $PWD/aloha-core/src -name 'fizz_buzzs.proto')\
>> +  --vw_labeled /tmp/dataset.vw
>> +```
>
>```
>def buildSpec[T <: GeneratedMessage : RefInfo, S <: RowCreator[T]](
>tempDir: Option[File], producers: List[RowCreatorProducer[T, S]],
>jsonSpec: String): Try[S] = {
>    logInfo("Compiling Aloha spec file")
>    val plugin = CompiledSemanticsProtoPlugin[T]
>    val compiler = TwitterEvalCompiler(classCacheDir = tempDir)
>    val semantics = CompiledSemantics(compiler, plugin, Nil)
>    val specBuilder = RowCreatorBuilder(semantics, producers)
>    specBuilder.fromString(jsonSpec)
>  }
>```
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/eHarmony/aloha/pull/47/files#r39460427